docs/utils/data.rst

```diff
@@ -221,3 +220,32 @@ the Internet at all.
 If you have a particular URL that is giving you trouble, you can download it
 using some other tool (e.g., ``wget``), possibly on another machine, and
 then use `~astropy.utils.data.import_file_to_cache`.
+
+Astropy Data and Clusters
+=========================
+
+Astronomical calculations often require the use of a large number of different
+processes on different machines with a shared home filesystem. This can pose
+certain complexities. In particular, if the many different processes attempt to
+download a file simultaneous this can overload a server or trigger security
+systems. The parallel access to the home directory can also trigger concurrency
+problems in the Astropy data cache, though we have tried to minimize these. We
+therefore recommend the following guidelines:
+
+ * Write a simple script that sets ``astropy.utils.iers.conf.auto_download = True``
+   and then accesses all cached resources your code will need, including source name
```

Comment:
```suggestion
   and then access all cached resources your code will need, including source name
```

Reply:
This is plural because the script should do all those things. Obviously I should rewrite to be clearer.

Reply:
"access" is a verb here right? Does the plural rule apply? I guess I am confused on English grammar again.

Reply:
Structurally it's "Write a script that accesses all resources". Sorry that's not actually a plural that's one of English's few verb conjugations. English is such a mess.

Reply:
Ah, okay. Makes sense now. Thanks for the clarification!